### PR TITLE
Updating gazebo 7 and 8

### PR DESCRIPTION
**Pull request by nkoenig**

### CODE REVIEW

File: gazebo/gazebo7/gzweb7/Dockerfile [hunk 1]
```diff
@@ -17,13 +17,12 @@ RUN apt-get update && apt-get install -q -y \
     nodejs-legacy \
     npm \
     pkg-config \
-    psmisc \
-    xvfb\
```
Comment (scpeters) [inline gazebo/gazebo7/gzweb7/Dockerfile]: `xvfb` is removed from this gzweb Dockerfile

File: gazebo/gazebo8/gzweb8/Dockerfile [hunk 2]
```diff
@@ -17,12 +17,13 @@ RUN apt-get update && apt-get install -q -y \
     nodejs-legacy \
     npm \
     pkg-config \
-    psmisc\
+    psmisc \
+    xvfb\
```
Comment (scpeters) [inline gazebo/gazebo8/gzweb8/Dockerfile]: `xvfb` is added to this `gzweb` Dockerfile

**Comment by ruffsl** [comment]
I pushed more changes to this branch to include a version bump for gazebo 8.
When making changes to auto generated dockerfiles, it's best to use the generation scripts so that template can update all the timestamps and check the repo index for you:
https://github.com/osrf/docker_images/tree/master/docker

If you would like, I could start the PR to the official library, or you may like to if you'd like to learn how:
https://github.com/osrf/docker_images/blob/master/doc/maintaining_repo.md#update-library
Its not much more than somthing like this:
https://github.com/docker-library/official-images/pull/2988

File: gazebo/gazebo7/gzweb7/Dockerfile (hunk 1 above)
Comment (ruffsl) [inline gazebo/gazebo7/gzweb7/Dockerfile]: Thanks, looks like gzweb's yaml wasn't updated to include the add packages we tried to solve:
https://github.com/osrf/docker_images/issues/43

**Comment by ruffsl** [comment]
Just noticed, @j-rivero why does `gzserver8` install `lsb-release`, but previous version don't?

**Comment by j-rivero** [comment]
> Just noticed, @j-rivero why does gzserver8 install lsb-release, but previous version don't?

It's mainly a difference between the [ubuntu core tarballs](https://github.com/tianon/docker-brew-ubuntu-core) used to generate the dockerfiles. The [trusty one has the lsb_release package](https://partner-images.canonical.com/core/trusty/current/ubuntu-trusty-core-cloudimg-amd64.manifest) while the [xenial one](https://partner-images.canonical.com/core/xenial/current/ubuntu-xenial-core-cloudimg-amd64.manifest) has not.

It's probably part of the effort to reduce the size of the base image.

```
jrivero@nium ~ $ docker run -i -t ubuntu:trusty lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 14.04.5 LTS
Release:	14.04
Codename:	trusty

jrivero@nium ~ $ docker run -i -t ubuntu:xenial lsb_release -a
exec: "lsb_release": executable file not found in $PATH
docker: Error response from daemon: Container command not found or does not exist..
```

**Comment by ruffsl** [comment]
Ok, I updated the template to clean lsb-release command dependency: https://github.com/osrf/docker_templates/pull/16
The gzweb build for 7 and 8 has also been updated.

**Comment by ruffsl** [comment]
@nkoenig , will you being make the PR change to the official library?
@mikaelarguedas may make one soon, so he could add this into his for you.
I'd just make a note about the use of `/etc/os-release` `/etc/lsb-release` to ask the maintainers if that's a stable distro lookup method.

**Pull request merged**